src/nxdoc/nuxeo-server/upgrading-the-nuxeo-platform/upgrade-from-lts-2016-to-93.md

```diff
@@ -85,6 +90,8 @@ tree_item_index: 96
 
 {{{multiexcerpt 'upgrade-9.2-sql-template-required' page='Upgrade from LTS 2016 following Fast Tracks'}}}
 
+{{{multiexcerpt 'upgrade-9.3-behavior.sqldirectories' page='Upgrade from LTS 2016 following Fast Tracks'}}}
```

Comment:
I think we need to review this whole Directories section. 
- “upgrade-9.2-sql-template-required” seems only required if you upgrade to 9.2, but not if you upgrade directly to 9.3 thanks to NXP-22785.
- We can remove “upgrade-9.3-behavior.sqldirectories”, but maybe provide the link to JIRA or the contribution in section “Using MongoDB with SQL directories”. Or maybe we don’t use excerpt for this part if doesn’t make sense.
WDYT?